src-os/runtime.js

```diff
@@ -0,0 +1,44 @@
+//Provides: ocaml_mtime_clock_period_ns
+function ocaml_mtime_clock_period_ns(_unit) {
+  return 0;
+}
+
+//Provides: mtime_clock_now
+//Requires: caml_int64_of_float
+//Requires: caml_raise_sys_error
+var performance_now;
+function test_perf_now(obj) {
+  if(obj && obj.performance && typeof obj.performance.now == "function"){
+    performance_now = obj.performance.now
+  }
+}
+test_perf_now(globalThis);
+if(performance_now === undefined)
+  test_perf_now(globalThis.perf_hooks)
+if(performance_now === undefined && typeof require == "function"){
+  var ph = require("perf_hooks");
+  test_perf_now(ph);
+}
+if(performance_now === undefined)
+  performance_now = (function () { caml_raise_sys_error("performance.now () is not available"); })
+function mtime_clock_now(){
+  var now_ms = performance_now ();
+  var now_ns = caml_int64_of_float(now_ms * 1e6);
```

Comment:
The original code had a more elaborate [conversion procedure](https://github.com/dbuenzli/mtime/blob/f472ffdf8bf2d3f941980d63073025df847c1173/src-jsoo/mtime_clock.ml#L34-L60). Did you find it unnecessary ?

Reply:
(Actually it's not: roll over it's: becoming imprecise).

Reply:
> I didn't spend much time on this.

I can see that ;-)

In fact your code doesn't work at all in the browser, you can't call `performance.now` directly as a function. If you checkout the [previous code](https://github.com/dbuenzli/mtime/blob/a2ab7d45debabca0a86fdbb73a3dde742c935a99/src-jsoo/mtime_clock.ml#L15-L32) it was a method call. e3b263b4bbdb179f605ac fixes it.